File: 13katlari.c
#include<stdio.h>

int main(){


	for(int i=100;i>0;i--){ //100 den 0 a kadar olan sayilar
	
		if(i%13==0){ //tam bolunuyor mu kontrol ediyoruz
			printf("%d\n",i);
		}
	}


	return 0;
}

File: 13Katlari.cs
using System;

namespace 13Katlari
{
    class Program
    {

        static void Main(string[] args)
        {
            //For Döngüsü ile 100'den 0 a kadar i'ye değer atandı.
            for (int i = 100; i >= 0; i--)
            {
                //%13 yazarak i 'nın modunu alındı. Yani  eğer sayı 13'e tam bölüne biliyorsa(13'ün katlarıysa) ekrana döndüdeki şartlar bitene kadar i 'nin değerini döndürür.
                if (i % 13 == 0)
                    Console.WriteLine(i);
            }

        }
    }
}
